src/jti_background.js

```diff
@@ -44,7 +44,8 @@ if (browserType !== 'Firefox') {
 // -------------------------------------------------------------------------- //
 
 var StorageID = 'Jira-Template-Injector';
-var emptyData = {'options': {'limit': []}, 'templates': {}};
+var rateStatus = 'JTI-Rate-Status';
+var emptyData = {'options': {'limit': []}, 'templates': {}, 'status': {}};
```

Comment:
I think maybe 'toggles' over 'status' so the format would be something like 
`'toggles' : { 'rateClicked': boolean, ... } `